src/javascript/app/base/header.js

```diff
@@ -547,21 +541,74 @@ const Header = (() => {
         Client.sendLogoutRequest();
     };
 
+    const bindHeaders = () => {
+        const high_risk_accounts_accordion_header = document.getElementById('high_risk_accounts');
+        const low_risk_non_eu_accordion_header = document.getElementById('low_risk_accounts_non_eu');
+        const low_risk_eu_accordion_header = document.getElementById('low_risk_accounts_eu');
+        const low_risk_eu_container = document.getElementById('account__switcher-accordion-eu');
+
+        if (Client.isHighRisk() || isEuCountry()) {
+            high_risk_accounts_accordion_header.style.display = 'flex';
+            low_risk_non_eu_accordion_header.style.display = 'none';
+            low_risk_eu_accordion_header.style.display = 'none';
+            low_risk_eu_container.style.display = 'none';
+        } else if (Client.isLowRisk()) {
+            high_risk_accounts_accordion_header.style.display = 'none';
+            low_risk_non_eu_accordion_header.style.display = 'flex';
+            low_risk_eu_accordion_header.style.display = 'flex';
+        }
+    };
```

Comment:
```suggestion
    const bindHeaders = () => {
        const high_risk_accounts_accordion_header = document.getElementById('high_risk_accounts');
        const low_risk_non_eu_accordion_header      = document.getElementById('low_risk_accounts_non_eu');
        const low_risk_eu_accordion_header              = document.getElementById('low_risk_accounts_eu');
        const low_risk_eu_container                             = document.getElementById('account__switcher-accordion-eu');

        if (Client.isHighRisk() || isEuCountry()) {
            high_risk_accounts_accordion_header.style.display = 'flex';
            low_risk_non_eu_accordion_header.style.display      = 'none';
            low_risk_eu_accordion_header.style.display              = 'none';
            low_risk_eu_container.style.display                             = 'none';
        } else if (Client.isLowRisk()) {
            high_risk_accounts_accordion_header.style.display = 'none';
            low_risk_non_eu_accordion_header.style.display      = 'flex';
            low_risk_eu_accordion_header.style.display               = 'flex';
        }
    };
```
@henry-deriv  we have convention in binary-static project to align lines by the farthest - sign. Please don't just apply this change but instead align them in IDE ) question: is it better to use `setVisibility(0)` instead on display none?

Reply:
yup, changed to visibility, also fixed code indentation to convention.